package.json

```diff
@@ -52,7 +52,9 @@
 	},
 	"dependencies": {},
 	"devDependencies": {
+		"cldr-data": ">=26",
```

Comment:
Answering the first questions, although I see your line of thought is going somewhere else already...

> @rxaviers is there any way to get an English-only cldr-data, since that's all we care about here (I think)?

No, cldr-data allows you to choose one of the coverage options given by CLDR (for instance, `modern` or `full` only).